src/Workspaces/CSharp/Portable/Formatting/Rules/IndentBlockFormattingRule.cs

```diff
@@ -45,6 +47,23 @@ private void AddTypeParameterConstraintClauseOperation(List<IndentBlockOperation
             }
         }
 
+        private void AddSwitchExpressionIdentationOperation(List<IndentBlockOperation> list, SyntaxNode node)
+        {
+            var switchExpression = node as SwitchExpressionSyntax;
+            if (switchExpression == null)
+            {
+                return;
+            }
+
+            var arms = switchExpression.Arms;
+            var startToken = arms.First().GetFirstToken(includeZeroWidth: true);
+            var endToken = arms.Last().GetLastToken(includeZeroWidth: true);
+            var span = CommonFormattingHelpers.GetSpanIncludingTrailingAndLeadingTriviaOfAdjacentTokens(startToken, endToken);
+
+            AddIndentBlockOperation(list, switchExpression.OpenBraceToken, startToken, endToken, IndentBlockOption.RelativeToFirstTokenOnBaseTokenLine);
```

Comment:
do you want to really force this? that means users can't put it anywhere else?

Reply:
Yes, I think so. The switch expression should be formatted strictly, like a switch statement (except a bit simpler).